pywal/__main__.py

```diff
@@ -15,11 +15,22 @@
 import shutil
 import sys
 
-try:
-    import colorama
-    colorama.just_fix_windows_console()
-except ImportError:
-    colorama = None
+if sys.platform.startswith("win"):
+    try:
+        import colorama
+        colorama.just_fix_windows_console()
+    except ImportError:
+        no_colorama_terms = [
+                "wezterm",
+                "alacritty",
+                "hyper",
+                "putty",
+                "ghostty",
+                "mintty",
+                ]
+        winterm = os.environ.get("TERMINAL")
```

Comment:
I think I would check TERM instead of TERMINAL.

Here is what mintty returns for OS vars:

```
$ set |rg TERM
TERM=xterm
TERM_PROGRAM=mintty
TERM_PROGRAM_VERSION=3.7.0
```

TERM is standard, I do not think TERMINAL is but I'm not 100% sure. I'm not sure this would work for mintty.

That being said, the worst that can happen is a warning message is displayed :-)

Reply:
`TERM` is standar but the issue is that term only represents the "compatibility level" of a terminal emulator, in this case mintty is saying that it provides the `xterm` default level, xterm is an actual program that has existed since 1984, the default terminal emulation level of xterm is that of a vt100 with some vt101 and vt102 features.

so no `TERM` is not useful for this, `TERM_PROGRAM` on the other hand will be